tests/php/sync/test_class.jetpack-sync-plugins-updates.php

```diff
@@ -12,9 +12,17 @@ class WP_Test_Jetpack_Sync_Plugins_Updates extends WP_Test_Jetpack_Sync_Base {
 	public function setUp() {
 		parent::setUp();
 
-
+		require ABSPATH . 'wp-includes/version.php';
 		if ( defined( 'PHP_VERSION_ID' ) && PHP_VERSION_ID < 50300 ) {
-			$this->markTestIncomplete("Right now this doesn't work on PHP 5.2");
+			$this->markTestIncomplete( "Right now this doesn't work on PHP 5.2");
```

Comment:
Since you added a space before the first `"`, maybe you want to add one at the end as well?